src/SmartEntity/SmartEntity.php

```diff
@@ -171,7 +171,8 @@ protected function add(string $property, array $arguments)
     {
         $this->fetchValue($arguments, $property);
 
-        if (!array_key_exists($property, $this->properties) ||
+        if (
+           !array_key_exists($property, $this->properties) ||
```

Comment:
```suggestion
            !array_key_exists($property, $this->properties) ||
```
Same as above, mea culpa